source/_components/gc100.markdown

```diff
@@ -0,0 +1,31 @@
+---
+layout: page
+title: "gc100"
+description: "Instructions on how to integrate gc100 with Home Assistant."
+date: 2017-10-26 17:20
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Hub
+ha_release: 0.57
+---
+
+[gc100](https://www.globalcache.com/products/gc-100/) can be integrated into Home Assistant. gc100 is a TCP-controllable
```

Comment:
How about:
```
The Global Caché [GC-100](https://www.globalcache.com/products/gc-100/) can be integrated into Home Assistant. The GC-100 is a TCP-controllable...
```
This would make it clear that this component is about a product from Global Caché. I suggest also using uppercase **GC-100** throughout to be consistent with the manufacturer's branding (except if talking about the Home Assistant component `gc100`).